### PR TITLE
Separate out general and moose specific installation instructions

### LINKED ISSUE #15209: Generalize conda installation documentation
## Reason
Including only general portions of the new conda installation documentation pages with regex expressions creates documentation build errors in downstream apps: ambiguous references to getting_started/index.md in dependent apps

## Design
Separating out the general conda installation instructions from the MOOSE specific information (e.g. links back to the MOOSE getting started pages)

## Impact
Easier linking between MOOSE and apps to ensure consistency in getting started installation instructions

### PR DESCRIPTION
**Pull request by sapitts**
Will allow for easier inclusion of the general information in downstream apps (and have tested with BISON)

Closes #15209

### CODE REVIEW

File: modules/doc/content/getting_started/installation/conda.md [hunk 1]
```diff
@@ -1,86 +1,23 @@
 # Conda MOOSE Environment
 
-Our preferred method for obtaining libraries necessary for MOOSE based Application development, is via Conda's myriad array of libraries. Follow these instructions to create an environment on your machine using Conda. At this time, Windows is not supported. Effort is being made towards this direction. But it is some time out.
+Our preferred method for obtaining libraries necessary for MOOSE based
+Application development, is via Conda's myriad array of libraries. Follow these
```
Comment (tophmatthews) [inline modules/doc/content/getting_started/installation/conda.md:4]: Is BISON being tested against CONDA?

File: modules/doc/content/getting_started/installation/conda.md [hunk 2]
```diff
@@ -1,86 +1,23 @@
...
+instructions to create an environment on your machine using Conda. At this time,
+Windows is not supported. Effort is being made towards this direction. But it is
```
Comment (tophmatthews) [inline modules/doc/content/getting_started/installation/conda.md:6]: very minor, but while I'm looking at this, maybe Link to Windows directions, and comma between the two sentences?

**Comment by sapitts** [comment]
@milljm this change made it through the documentation build, and the couple of pages on which I moved stuff around look the same as from your original merge request. Would you review this please?

File: modules/doc/content/getting_started/installation/conda.md (hunk 1 above)
Comment (sapitts) [reply modules/doc/content/getting_started/installation/conda.md:4]: I'm not sure about BISON specifically, but MOOSE is (although Jason will know).  The old MOOSE package links that we linked to in BISON are no longer available, so the MOOSE team recommended updating the getting started instructions in BISON.

Comment (sapitts) [reply modules/doc/content/getting_started/installation/conda.md:4]: Yes, BISON is tested against conda on the Mac test of the civet recipe

File: modules/doc/content/getting_started/installation/conda.md (hunk 2 above)
Comment (milljm) [reply modules/doc/content/getting_started/installation/conda.md:6]: I agree with the comma separation. My grammar skills have always been found wanting.

File: modules/doc/content/getting_started/installation/conda.md [hunk 3]
```diff
@@ -1,86 +1,23 @@
...
-The MOOSE team will make periodic updates to the conda packages. To stay up-to-date, activate the moose environment, and perform an update:
-
-```bash
-conda activate moose
-conda update --all
-```
-
-!alert note title= sudo is not necessary
-If you find yourself applying the use of `sudo` for any of the above conda commands... something's not right. The most common reason for needing sudo, is due to an improper installation. Conda *should* be installed to your home directory, and without any use of `sudo`.
+!include installation/install_miniconda.md
```
Comment (milljm) [inline modules/doc/content/getting_started/installation/conda.md:15]: I'm a big fan of creating an include here... Nice.

File: modules/doc/content/getting_started/installation/conda.md [hunk 4]
```diff
@@ -1,86 +1,23 @@
...
 
-## Uninstall Conda MOOSE Environment
-
-If you wish to remove the moose environment at any time, you may do so using the following commands:
-
-```bash
-conda deactivate   # if 'moose' was currently activated
-conda remove --name moose --all
-```
+!include installation/uninstall_conda.md
```
Comment (milljm) [inline modules/doc/content/getting_started/installation/conda.md:23]: 👍

File: modules/doc/content/getting_started/installation/windows10.md [hunk 5]
```diff
@@ -11,51 +11,7 @@ Caveats:
...
-```
-
-## Configure WSL to use VcXsrv
-
-Modify your bash profile to allow WSL to connect to VcXsrv.
-
-```bash
-echo "export DISPLAY=localhost:0" >> ~/.bashrc
-```
+!include installation/wsl.md
```
Comment (milljm) [inline modules/doc/content/getting_started/installation/windows10.md:14]: Nice. Considering I see a native windows build on the horizon.

**Review by milljm — DISMISSED** [review]
This all looks nice. I agree with Topher about the comma separation. Minor, and up to you if you wish to make that change. Otherwise, I approve.

File: modules/doc/content/getting_started/installation/windows10.md (hunk 5 above)
Comment (sapitts) [reply modules/doc/content/getting_started/installation/windows10.md:14]: yeah..I asked @dschwen if he wanted to update the wsl instructions and he had to point out that wsl and windows are not exactly the same thing to me....but now I've got my bases covered in the file names :-)

**Review by milljm — APPROVED** [review]
👍

**Comment by sapitts** [comment]
Thanks for the merge!

**Pull request merged**